integration/index.ts

```diff
@@ -40,15 +40,16 @@ export default function NetlifyCMS({
             ],
           },
         };
+
         updateConfig(newConfig);
 
-        injectScript(
+       /*  injectScript(
```

Comment:
I don’t think we should comment this out entirely until we have a replacement along the lines of what you suggested in #14. Instead:

1. Add a `disableIdentityWidgetInjection` option to `NetlifyCMSOptions` (L9-17)
2. Read that flag in the `NetlifyCMS` function
3. Use an `if (!disableIdentityWidgetInjection)` statement around the `injectScript`

Then a user would use the integration passing `disableIdentityWidgetInjection: true` as an option.